olm-catalog/serverless-operator/project.yaml

```diff
@@ -1,75 +1,75 @@
+    # is ignored as it is overridden by fake version via KUBERNETES_MIN_VERSION.
+    # This value is used for CSV's min version validation.
+    minVersion: 1.25.0
+  golang: '1.22'
+  nodejs: 20.x
+  ocpVersion:
+    list:
+      - "4.13"
+      - "4.14"
+      - "4.15"
+      - "4.16"
+      - "4.17"
+    min: '4.13'
+    max: '4.17'
+    label: 'v4.13'
 dependencies:
-    serving: knative-v1.15
-    # serving midstream branch name
-    serving_artifacts_branch: release-v1.15
-    # versions for networking components
-    kourier: knative-v1.15
-    net_kourier_artifacts_branch: release-v1.15
-    net_istio: knative-v1.15
-    net_istio_artifacts_branch: release-v1.15
-    redhat-knative-istio-authz-chart: 1.35.0
-    maistra: 2.4-latest
-    eventing: knative-v1.15
-    # eventing core midstream branch name
-    eventing_artifacts_branch: release-v1.15
-    # eventing-kafka-broker promotion tag
-    eventing_kafka_broker: knative-v1.15
-    # eventing-kafka-broker midstream branch or commit
-    eventing_kafka_broker_artifacts_branch: release-v1.15
-    # eventing-istio promotion tag
-    eventing_istio: knative-v1.15
-    # eventing-istio midstream branch or commit
-    eventing_istio_artifacts_branch: release-v1.15
-    # backstage-plugins promotion tag
-    backstage_plugins: knative-v1.15
-    # backstage-plugins midstream branch or commit
-    backstage_plugins_artifacts_branch: release-v1.15
-    cli: knative-v1.15
-    func:
-        promotion_tag: knative-v1.15
-        tekton_s2i: registry.redhat.io/source-to-image/source-to-image-rhel8@sha256:6a6025914296a62fdf2092c3a40011bd9b966a6806b094d51eec5e1bd5026ef4
-        tekton_buildah: registry.redhat.io/rhel8/buildah@sha256:3d505d9c0f5d4cd5a4ec03b8d038656c6cdbdf5191e00ce6388f7e0e4d2f1b74
-        nodejs_20_minimal: registry.access.redhat.com/ubi8/nodejs-20-minimal@sha256:a2a7e399aaf09a48c28f40820da16709b62aee6f2bc703116b9345fab5830861
-        openjdk_21: registry.access.redhat.com/ubi8/openjdk-21@sha256:441897a1f691c7d4b3a67bb3e0fea83e18352214264cb383fd057bbbd5ed863c
-        python-39: registry.access.redhat.com/ubi8/python-39@sha256:27e795fd6b1b77de70d1dc73a65e4c790650748a9cfda138fdbd194b3d6eea3d
-    operator: 1.15.4
-    # Previous versions required for downgrade testing
-    previous:
-        serving: knative-v1.14
-        eventing: knative-v1.14
-        eventing_kafka_broker: knative-v1.14
-    mustgather:
-        image: quay.io/openshift-knative/must-gather
+  serving: knative-v1.15
+  # serving midstream branch name
+  serving_artifacts_branch: release-v1.15
+  # versions for networking components
+  kourier: knative-v1.15
+  net_kourier_artifacts_branch: release-v1.15
+  net_istio: knative-v1.15
+  net_istio_artifacts_branch: release-v1.15
+  redhat-knative-istio-authz-chart: 1.35.0
+  service_mesh_proxy: registry.redhat.io/openshift-service-mesh/proxyv2-rhel8@sha256:02b834fd74da71ec37f6a5c0d10aac9a679d1a0f4e510c4f77723ef2367e858a
```

Comment:
I assume that hash is equivalent with 2.4-latest?

Reply:
See discussion here: https://redhat-internal.slack.com/archives/CKR568L8G/p1727254666855229